### DISCUSSION TITLE
#semanticClimate Website Restructure

### DISCUSSION THREAD

**Discussion by ShweataNHegde** [petermr/petermr#71] [General]
# Styling Issues
- Hyperlinks don't show up. 

# Landing page
- We need a specific tagline
- Summarize the page
- What's #semanticClimate goes to the top
- More events

# About (should come next to home)
- Goals and how to get involved need expanding
- Teams page!
- We need more pictures

# Events
- Having a calendar might help. 
- Untangle events and blog page (not sure how)

# Participate page

# Volunteer!
Do we need this page? Makes sense to combine participate and volunteer page, perhaps

# Suggestions
- Teams page
- Toolchain page

## Toolchain page
- Python and Google Colab 101
- Intro to all our tools
- Other resources in the toolchain -- dictionaries, HTML, etc.

**Comment by ProbablyShubham** [comment]
**Getting Started** needs integration to other pages like where we talk about upcoming Hackathons

**Reply by ShweataNHegde** [reply]
# GY's Recommendations
- Ways to add more Menu sections (eg. dedicated pages for PEOPLE, Our partners, Intern Stories/Reports etc)
- Ways to add more pages (not just within Blog or Events, or other Menu sections - but as links from existing pages)
- How to edit Webpage Header / Footers
- Making it easier for folks to join us - Can we add the Induction Task somewhere? 
- How to Make the website interactive - eg. Can we add A Map showing our locations, or our events, a dynamic one that changes with time? 
- Creating handles for outreach - can we add widgets on who has been visiting etc?
- FSCI idea- Creating ways for folks to join/run our Hackathon asynchronously, using all our existing material- The way we're doing for FSCI just now!

**Comment by ShweataNHegde** [comment]
## New Hierarchy for Nav Bar
- Home 
     - What's #sC? 
     - What we do?
     -  CTA
     - Links to Events, Blogs, Teams, etc
- About
    - More description about what we do, goals and how we work
    - Pictures
- Team 
- Events Page
    - Current
    - Past
- Events Report Page
- Tools
- Blog 
- Participate/Volunteer and Contact

**Comment by ShweataNHegde** [comment]
2023-08-04

## Priority list
- Teams (Content: @Waheb and @Renu; Tech: @axel and @shweata)
    - Name
    - Affiliation
    - Location
    - Role
    - Duration
    - Biography 
    - Photograph
    - Their video
    - Social media/Email/ORC ID (People's preference)
- About -> What we do, how we do, ... (Tech: @axel and @shweata)
- Tutorials (content: ?)
- Homepage
     - Add Twitter stream to the homepage
     - Contact (Google Form @shubham)
- Make playlists and feature one video on homepage
- Have blog posts that doesn't show up on the blog stream
- Add GDPR  to footer (@axel)

**Reply by xldrkp** [reply]
I fixed GDPR in the footer and added "About" without removing it from the top nav. Important: To have the links work from every page, it needs to be an absolute path like `/p/en/gdpr`. This will show errors when run locally on your computer.

**Reply by xldrkp** [reply]
Added the feature to hide posts with `hide: true` in the front matter. Cf. an example at https://semanticclimate.org/p/en/posts/testpost/. This page stems from the file `en/posts/testpost.md`. Also cf. the information on how that is done *on* the testpost page and in the YAML frontmatter of the related file.

**Reply by xldrkp** [reply]
Found https://blog.hubspot.com/marketing/creative-agency-team-pages as some inspiration for the team page.

**Comment by ProbablyShubham** [comment]
### Necessary Changes Needed on the Website: As Discussed with GY

- [ ] On the [Participate! (docs) page](https://semanticclimate.org/p/en/docs/), create a section for the [City Climate Plans](https://semanticclimate.org/city-climate-plans/#H9711914), similar to the one made for the Alpha Tests.
- [ ] On the same page, the order of arrangement of the three sections should be in alphabetical order, as follows: _**Alpha tests**_ followed by _**Build a Climate Explorer**_, followed by _**City Climate Plans**._
- [ ] The [#semanticClimate Twitter feed ](https://twitter.com/semanticClimate) needs to be embedded somewhere onto the [Events page](https://semanticclimate.org/p/en/events/). 
- [ ] The Teams page(s) are being built by @Waheb, which will later have to be incorporated on the website.
